src/main/java/org/folio/rest/impl/ProxiesForAPI.java

```diff
@@ -197,7 +197,7 @@ public void getProxiesforById(String id,
     vertxContext.runOnContext(v -> {
       try {
         Criterion criterion = new Criterion(new Criteria().addField(ID_FIELD_NAME)
-          .setJSONB(false).setOperation("=").setValue("'" + id + "'"));
+          .setJSONB(false).setOperation("=").setVal(id));
```

Comment:
Can you replace the complete method by a PgUtil.getById invokation so that this duplicated code gets removed?